fe/fe-core/src/main/java/org/apache/doris/nereids/rules/expression/rules/FoldConstantRuleOnFE.java

```diff
@@ -405,6 +405,11 @@ public Expression visitOr(Or or, ExpressionRewriteContext context) {
 
     @Override
     public Expression visitLike(Like like, ExpressionRewriteContext context) {
```

Comment:
on 2.0, i think it's more important to investigate why c1 like '%xxx%' can't infer out c1 is not null by foldOnBe, than support like folding on fe.

Reply:
yes, but i think we can fix this problem first by adding this feature to fe folding constant

Reply:
ok